packages/UsageAnalysis/files/Test Track/Viewers/Correlation plot.md

```diff
@@ -0,0 +1,24 @@
+### Correlation plot
+
+1. Open the **SPGI dataset**.
```

Comment:
specify how

Reply:
done
Open the **SPGI dataset** by pressing 'star' icon in TestTrack (with tooltip "Open test data").